lib/brainstem/api_docs/formatters/markdown/presenter_formatter.rb

```diff
@@ -169,40 +157,39 @@ def format_sort_orders!
             end
           end
 
-
           def format_associations!
-            if presenter.valid_associations.any?
-              output << md_h5("Associations")
-
-              output << "Association Name | Associated Class | Description\n"
-              output << " --------------  |  --------------  |  ----------\n"
-
-              output << presenter.valid_associations.inject("") do |buffer, (_, association)|
-                link = presenter.link_for_association(association)
-                if link
-                  link = md_a(association.target_class, link)
-                else
-                  link = association.target_class.to_s
-                end
+            return if presenter.valid_associations.empty?
 
-                desc = association.description.to_s
-                if association.options && association.options[:restrict_to_only]
-                  desc += "." unless desc =~ /\.\s*\z/
-                  desc += "  Restricted to queries using the #{md_inline_code("only")} parameter."
-                  desc.strip!
-                end
+            output << md_h5("Associations")
 
-                buffer << md_inline_code(association.name) + " | " + link + " | " + desc + "\n"
+            output << "Association Name | Associated Class | Description\n"
+            output << " --------------  |  --------------  |  ----------\n"
+
+            output << presenter.valid_associations.inject("") do |buffer, (_, association)|
+              link = presenter.link_for_association(association)
+              if link
+                link = md_a(association.target_class, link)
+              else
+                link = association.target_class.to_s
               end
 
-              output << "\n"
+              desc = association.description.to_s
+              if association.options && association.options[:restrict_to_only]
+                desc += "." unless desc =~ /\.\s*\z/
+                desc += "  Restricted to queries using the #{md_inline_code("only")} parameter."
+                desc.strip!
+              end
+
+              buffer << md_inline_code(association.name) + " | " + link + " | " + desc + "\n"
```

Comment:
Aside: it's kinda weird to use `inject` as a way to mutate the `buffer` string. I would assume we'd use a `map` if we wanted a functional approach. But mutating `output` is also fine (and perhaps more straightforward?)

Reply:
Yeah. We followed this pattern all over the code so continued it